### ISSUE TITLE
member can't be removed from group

### ISSUE THREAD

**Issue by computersalat** [signalapp/Signal-Android#8586]
- [X] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [X] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Added user to group cvan't be removed

### Steps to reproduce
- create group
- add memeber to group
- try to remove member again

**Actual result:** member can't be removed.
**Expected result:** member should be removable

### Screenshots
You can't do screenshots in signal.

### Device info

**Device:** OnePlus
**Android version:** any
**Signal version:** latest (4.32.8)

### Link to debug log
how to debug ?

**Comment by computersalat** [comment]
that is bad ... so you can add one to a group but you can't remove again
very clever ...

**Comment by computersalat** [comment]
hence I guess there is also no chance to delete a group ... cause all members need to leave first ...

**Comment by computersalat** [comment]
... then none is over to delete the group.
again very clever ...

**Comment by computersalat** [comment]
why not having a 'decentralized' group state ?

**Comment by natrius** [comment]
Maybe add this somehwere https://support.signal.org/hc/en-us/ here as i found some issues related to this question. Explain there that it is a feature and not a bug or better, it is intended. (Yeah, old issue, but imho no need for a new issue)

**Comment by mlncn** [comment]
This comes as quite a shock when you need it— such as when your friendly neighborhood police department is rampaging through your city, and someone in your group get arrested for three days for being on their front porch during curfew, and they therefore cannot remove themselves from the group, but have in an abundance of caution asked that they be removed. 

Agree strongly with the requests to document very clearly this fact, and not just in closed issues!

But moreover, i think there should be a feature request open for this.  There could be three kinds of groups, that can be chosen at the start:

  * Fixed:  No one can add or remove from the initial members (except removing oneself).
  * Open:  Any member can add members; members can only remove themselves.
  * Melee:  Any member can add *or* remove members.

Yeah, the names need work, but you get the idea.  This gives the initiator some choices without the need for the concept of group admins.

Anyhow here's the same issue, now locked, opened for iOS:  https://github.com/signalapp/Signal-iOS/issues/1621

**Comment by alan-signal** [comment]
@mlncn It doesn't help, but we're working on it https://signal.org/blog/signal-private-group-system/

The solution for now is to open a new group with everyone except the member you want to eject.

**Comment by jeroenpraat** [comment]
> @mlncn It doesn't help you now, but we're working on it https://signal.org/blog/signal-private-group-system/

@alan-signal Any update about this? I really need it. Here Telegram has much of a lead, but Telegram has another issues as we know.

**Comment by alan-signal** [comment]
No, you might want to look at the forum for updates and progress signs https://community.signalusers.org/
